src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -382,6 +389,7 @@ public void validateSectionsAndTeams(List<StudentAttributes> studentList, String
     
     /**
      * Validates teams for any team name violations
```

Comment:
It's a good practice to end the first sentence (usually Javadoc summary) with period.